starport/services/chain/plugin-stargate.go

```diff
@@ -88,15 +87,15 @@ func (p *stargatePlugin) ShowAccountCommand(accountName string) step.Option {
 	)
 }
 
-func (p *stargatePlugin) ConfigCommands() []step.Option {
+func (p *stargatePlugin) ConfigCommands(_ string) []step.Option {
 	return nil
 }
 
-func (p *stargatePlugin) GentxCommand(conf starportconf.Config) step.Option {
+func (p *stargatePlugin) GentxCommand(chainID string, conf starportconf.Config) step.Option {
 	return step.Exec(
 		p.app.d(),
 		"gentx", conf.Validator.Name,
-		"--chain-id", p.app.n(),
+		"--chain-id", chainID,
 		"--keyring-backend", "test",
```

Comment:
Shouldn't we make this option configurable for the future?

Reply:
Uh, can you please elaborate? With this change, chain id already became configurable inside `config.yml` by `genesis.chain_id`.

Reply:
Hm, maybe you mean `keyring-backend`. Yep, we should make it configurable, it will specially be useful when we introduce a `start` command for production.